fs/sync/sync_test.go

```diff
@@ -595,6 +595,11 @@ func TestSyncAfterRemovingAFileAndAddingAFileSubDirWithErrors(t *testing.T) {
 		},
 		fs.Config.ModifyWindow,
 	)
+
+	accounting.Stats.ResetCounters()
+	fs.CountError(nil)
+	fs.Config.IgnoreErrors = true
+	require.NoError(t, Sync(r.Fremote, r.Flocal))
```

Comment:
I suggest you make a new test for this - you need to check the files got deleted, because currently your code doesn't delete them!

Note you need to reset fs.Config.IgnoreErrors at the end of the test in a `defer` statement - there are lots of examples in this file.